src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Drawable, AreaCalculable {
+    private double sideA;
+    private double sideB;
+    private double height;
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double height, String color) {
+        super.setColor(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.height = height;
+    }
+
+    @Override
+    public double area() {
+        return (sideA + sideB) * height / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isoscelesTrapezoid, area:" + area() + "sq. units, side: "
```

Comment:
When overriding the 'draw' method, ensure that the output message is consistent and clear. The message should include commas to separate the figure's properties for better readability. Additionally, 'isoscelesTrapezoid' should start with a capital letter since it's the name of the figure.